packages/angular-ui-components/package.json

```diff
@@ -25,49 +25,55 @@
 	"scripts": {
 		"clean": "rm -rf node_modules dist",
 		"start": "ng serve",
-		"build": "ng build --prod",
+		"build": "ng build --configuration production",
 		"build:watch": "ng build --watch",
 		"test": "ng test",
 		"lint": "ng lint",
 		"postbuild": "node .scripts/copy-assets.js"
 	},
 	"peerDependencies": {
-		"@angular/common": ">=11.2.14",
-		"@angular/core": ">=11.2.14",
-		"rxjs": ">=6.6.0"
+		"rxjs": ">=7.5.0"
```

Comment:
To not mess with the configuration of `angular-client-sdk` we should maintain the rxjs, if possible.